cmd/dockerd/config.go

```diff
@@ -3,20 +3,14 @@ package main
 import (
 	"runtime"
 
-	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/daemon/config"
 	"github.com/docker/docker/opts"
-	"github.com/docker/docker/plugin/executor/containerd"
 	"github.com/docker/docker/registry"
 	"github.com/spf13/pflag"
 )
 
-const (
-	// defaultShutdownTimeout is the default shutdown timeout for the daemon
-	defaultShutdownTimeout = 15
-	// defaultTrustKeyFile is the default filename for the trust key
-	defaultTrustKeyFile = "key.json"
-)
+// defaultTrustKeyFile is the default filename for the trust key
+const defaultTrustKeyFile = "key.json"
```

Comment:
Keeping this one here for now, as this should go away soon (when we remove schema 2 v1 support)